packages/paste-design-tokens/tokens/aliases/font-family.yml

```diff
@@ -1,3 +1,7 @@
 aliases:
   font-family-10: "'Inter var experimental', 'Inter var', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif"
   font-family-20: "'Fira Mono', 'Courier New', Courier, monospace"
+  font-family-ja: "'Inter var experimental', 'Inter var', Hiragino Sans, 'ヒラギノ角ゴ ProN W3', 'Hiragino Kaku Gothic ProN', 'メイリオ', Meiryo, Osaka, 'MS PGothic', sans-serif"
+  font-family-ko: "'Inter var experimental', 'Inter var', 'Microsoft JhengHei', 微軟正黑體, 'Microsoft JhengHei UI', 'Microsoft YaHei', 微軟雅黑, 宋体, SimSun, sans-serif"
+  font-family-zh-traditional: "'Inter var experimental', 'Inter var', 'Microsoft JhengHei', 微軟正黑體, 'Microsoft JhengHei UI', 'Microsoft YaHei', 微軟雅黑, 宋体, SimSun, sans-serif"
```

Comment:
Question: the selector where this is used is `html:lang(zh-TW), html:lang(zh-HK)`. Is this an ok name? Or should it be something else?

Reply:
This is more to be applied to the tokens file, but we tend to favour verbosity where possible. It's just easier for everyone to understand when coming into the code base, even if you have no prior experience, when things are spelt out in full.

I wouldn't be ashame of 

```
font-family-japanese: foo;
font-family-korean: bar;
font-family-traditional-chinese: boo;
font-family-simplified-chinese: bam;
```

As the token names